portfolio/pom.xml

```diff
@@ -35,7 +45,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>google.com:lilhunt-step-intern</deploy.projectId>
```

Comment:
Can you please check with VSE if the GCP project ID should be committed in GitHub?

Reply:
Waiting for confirmation now

Reply:
Fine to include